packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterIndexComponent extends Component {
+  get termsUrl() {
```

Comment:
if we are going to have more than one of these arguments, maybe we can use `url` as prefix:
- `urlForTerms`
- `urlForSupport`
- `urlForPrivacy`
- etc.